admin/class-boldgrid-backup-admin-core.php

```diff
@@ -560,6 +568,12 @@ public function __construct() {
 
 		$this->pagenow = $pagenow;
 
+		// Instantiate Configs Array
+		$this->configs = Boldgrid_Backup_Admin::get_configs();
```

Comment:
Should the class `configs` and `plugin` class properties be defined in the top of the file?

Reply:
Yes they should be. Will adjust this

Reply:
Added plugin class property to the top of the files, but the configs definition was already there